sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/src/main/java/com/microsoft/opentelemetry/exporter/azuremonitor/AzureMonitorExporterBuilder.java

```diff
@@ -134,15 +136,32 @@ public AzureMonitorExporterBuilder configuration(Configuration configuration) {
     }
 
     /**
-     * Sets the instrumentation key to use for exporting telemetry events to Azure Monitor.
-     * @param instrumentationKey The instrumentation key of the Azure Monitor resource.
+     * The connection string to use for exporting telemetry events to Azure Monitor.
+     * @param connectionString The connection string for the Azure Monitor resource.
      * @return The updated {@link AzureMonitorExporterBuilder} object.
+     * @throws NullPointerException If the connection string is {@code null}.
+     * @throws IllegalArgumentException If the connection string is invalid.
      */
-    public AzureMonitorExporterBuilder instrumentationKey(String instrumentationKey) {
-        this.instrumentationKey = instrumentationKey;
+    public AzureMonitorExporterBuilder connectionString(String connectionString) {
+        this.instrumentationKey = extractValueFromConnectionString(connectionString, "InstrumentationKey");
+        this.endpoint(extractValueFromConnectionString(connectionString, "IngestionEndpoint"));
         return this;
```

Comment:
if the `IngestionEndpoint` key is absent, ideally should default to `https://dc.services.visualstudio.com`

Reply:
Updated to use the default endpoint if connection string doesn't have one. Although, I am curious when would a connectionString not have the IngestionEndpoint. If it doesn't have one, will that qualify as a valid connectionString?

Reply:
that's a good question, when we originally implemented connection strings, the Portal U/X only included IngestionEndpoint for gov clouds, so the default was important, but now the Portal U/X always includes IngestionEndpoint, so the default is likely not needed anymore. I'll start an internal thread and add you to see if we can find out.